<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>551</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="polyShape">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Polygon</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Images</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>256</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </widget>
    <widget class="QListView" name="ImgList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="WidgetImgList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="imgListSortAsc">
        <property name="text">
         <string>Sort Asc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgListSortDesc">
        <property name="text">
         <string>Sort Desc</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>591</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string>Image in use</string>
    </property>
    <widget class="QToolButton" name="zoomOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom out</string>
     </property>
    </widget>
    <widget class="QToolButton" name="zoomIn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>530</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom In</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>9</y>
      <width>101</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Annotation</string>
    </property>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>311</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>Redo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>551</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Save Annotaions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Load Annotaions</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>281</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <item>
          <widget class="QTextEdit" name="classNameField"/>
         </item>
         <item>
          <widget class="QPushButton" name="addClass">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="classList"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="loadClass">
             <property name="text">
              <string>Load Class</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteClass">
             <property name="text">
              <string>Delete Class</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="classSortAsc">
         <property name="text">
          <string>Sort Asc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="classSortDesc">
         <property name="text">
          <string>Sort Desc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>splitter_3</zorder>
   <zorder>splitter_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>graphicsView</zorder>
   <zorder>polyShape</zorder>
   <zorder>groupBox</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>classSortAsc</zorder>
   <zorder>classSortDesc</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBug_Report"/>
    <addaction name="actionFAQ"/>
    <addaction name="actionF_A_Q"/>
    <addaction name="actionSoftware_Information"/>
   </widget>
   <addaction name="menuImages"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionF_A_Q">
   <property name="text">
    <string>F.A.Q.</string>
   </property>
  </action>
  <action name="actionBug_Report">
   <property name="text">
    <string>Bug Report</string>
   </property>
  </action>
  <action name="actionSoftware_Information">
   <property name="text">
    <string>Software Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
